tests/util/translator/translator.spec.tsx

```diff
@@ -70,40 +75,33 @@ describe("Translator", () => {
 });
 
 // =============================================================================
-// MOCKS
+// MOCK ELEMENTS
 // =============================================================================
 
+const MOCK_TEXT_BASED_ELEMENT: TElementMap = {
+    ...generateMockElement({
+        type: EElementType.TEXT,
+        id: "mockId1",
+        internalId: "mockInternalId1",
+        requiredErrorMsg: "Input is required",
+    }),
+};
```

Comment:
no need to spread since the function is already generating a new object
we typically spread to do shallow cloning (you can read up more if you are unsure)

```suggestion
const MOCK_TEXT_BASED_ELEMENT: TElementMap = generateMockElement({
        type: EElementType.TEXT,
        id: "mockId1",
        internalId: "mockInternalId1",
        requiredErrorMsg: "Input is required",
    });
```